sdk-core-protos/protos/local/temporal/sdk/core/workflow_activation/workflow_activation.proto

```diff
@@ -316,6 +324,35 @@ message DoUpdate {
     bool run_validator = 7;
 }
 
+message ResolveNexusOperationStart {
+    // Sequence number as provided by lang in the corresponding ScheduleNexusOperation command
+    uint32 seq = 1;
+    oneof status {
+        // The operation started asynchronously. Contains an ID that can be used to perform
+        // operations on the started operation by, ex, clients. A `ResolveNexusOperation` job will
+        // follow at some point.
+        string operation_id = 2;
+        // The operation "started" but only because it's also already resolved. A
+        // `ResolveNexusOperation` job will be in the same activation.
+        google.protobuf.Empty started_sync = 3;
```

Comment:
Can we make this a bool so we don't have to bring in "empty"? Since we have source compat with lang on protos, we are safe changing this to a full message later, we don't need to be concerned with wire compat IMO.

Arguably we could just remove this. I can't see the situation where lang cares whether start and complete are in the same activation or not (same for child workflows). Probably same for the field below, but I guess too many fields doesn't technically hurt.

Reply:
Sure, it can be a bool